osu.Game/Skinning/LegacySkinConfiguration.cs

```diff
@@ -12,6 +14,13 @@ public class LegacySkinConfiguration : SkinConfiguration
         /// </summary>
         public decimal? LegacyVersion { get; internal set; }
 
+        public LegacySkinConfiguration()
+        {
+            // Roughly matches osu!stable's slider border portions.
+            // Can't use nameof(SliderBorderSize) as the lookup enum is declared in the osu! ruleset.
+            ConfigDictionary["SliderBorderSize"] = 0.77f.ToString(CultureInfo.InvariantCulture);
```

Comment:
I'm not sure I like the magic string too much here... Doing it "properly" would require letting the ruleset patch this onto the skin configuration, but that feels almost overengineered, and moving the lookups to the main assembly is also icky... I guess maybe we let this slide because legacy?

Reply:
Yeah, or reverting back to [what I had at first](https://github.com/ppy/osu/pull/10041/commits/e7eaaf8b02efd43448b6dbbb11b4e98e7062aca7) which doesn't include magic string or overrides, just an adjustment for any legacy slider body, whichever is fine for you.

Reply:
I will continue to insist that if we have a mechanism for looking up skin values, we should be using it. If it's not good enough for this use case, then that should be fixed.